components/collective-page/SectionHeader.js

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Info } from '@styled-icons/feather/Info';
+import { useIntl } from 'react-intl';
+import styled from 'styled-components';
+
+import i18nCollectivePageSection from '../../lib/i18n-collective-page-section';
+
+import Container from '../Container';
+import { Box, Flex } from '../Grid';
+import StyledHr from '../StyledHr';
+import StyledTooltip from '../StyledTooltip';
+import { P } from '../Text';
+
+import { Dimensions } from './_constants';
+import SectionTitle from './SectionTitle';
+
+import budgetSectionHeaderIcon from '../../public/static/images/collective-navbar-v2/CollectiveSectionHeaderIconBudget.png';
+import connectSectionHeaderIcon from '../../public/static/images/collective-navbar-v2/CollectiveSectionHeaderIconConnect.png';
+import contributeSectionHeaderIcon from '../../public/static/images/collective-navbar-v2/CollectiveSectionHeaderIconContribute.png';
```

Comment:
Let's avoid versioning ("v2") in folder names.

Also "navbar" doesn't seem the right name because these are section headers.